docs-beta/src/components/Illustration.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export const Illustration = (props) => {
+  const imageStyle = `mx-auto ${
+    props.isWave ? "w-full h-full" : "w-3/4 h-3/4 md:w-full md:h-full"
```

Comment:
When it is wave, trying also setting a max height, maybe something like max-h-9 which should be about 36px according to the docs... :)

https://tailwindcss.com/docs/max-height